textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/connector/OneToNBroadcastConnector.java

```diff
@@ -17,14 +17,18 @@
  */
 public class OneToNBroadcastConnector implements IConnector {
     
+    private static final int PRE_OPEN = -2;
+    private static final int OPENED = 0;
```

Comment:
Can we use "-1" for "OPENED" and "0" for "CLOSED" to be consistent with the order?  Or there is a special meaning for each number?